test/e2e/selectors/selectors_test.go

```diff
@@ -116,72 +116,160 @@ var _ = Describe("Field Selectors in Flight Control", func() {
 			deviceName := extractDeviceName()
 			startTimestamp, endTimestamp := generateTimestamps()
 			out, err := harness.CLI("get", "devices", "--field-selector",
-				fmt.Sprintf("metadata.creationTimestamp>=%s,metadata.creationTimestamp<%s", startTimestamp, endTimestamp))
+				fmt.Sprintf("metadata.creationTimestamp>=%s,metadata.creationTimestamp<%s", startTimestamp, endTimestamp), "-owide")
 			Expect(err).ToNot(HaveOccurred())
 			Expect(out).To(ContainSubstring(deviceName))
 		})
-
 	})
 
 	Context("Advanced Functionality Tests", Label("77947"), func() {
+
 		It("filters devices by multiple field selectors", func() {
 			deviceName := extractDeviceName()
 			startTimestamp, _ := generateTimestamps()
 			out, err := harness.CLI("get", "devices", "-l", "region=eu-west-1", "--field-selector",
-				fmt.Sprintf("metadata.creationTimestamp>=%s", startTimestamp))
+				fmt.Sprintf("metadata.creationTimestamp>=%s", startTimestamp), "-owide")
 			Expect(err).ToNot(HaveOccurred())
 			Expect(out).To(ContainSubstring(deviceName))
 		})
 
 		It("excludes devices by name", func() {
-			out, err := harness.CLI("get", "devices", "--field-selector", "metadata.name!=device1-name")
+			out, err := harness.CLI("get", "devices", "--field-selector", "metadata.name!=device1-name", "-owide")
 			Expect(err).ToNot(HaveOccurred())
 			Expect(out).ToNot(ContainSubstring("device1-name"))
 		})
 	})
 
-	Context("Negative Tests", Label("77948"), func() {
+	// Label Selectors Tests
+	Context("Label Selectors Tests", Label("78751"), func() { // Fixed parentheses
+		It("filters devices by region in a set", func() {
```

Comment:
Please, update It with By. As discussed previously, every It is appeared as separated TC